packages/lib/src/vis/hooks.ts

```diff
@@ -18,29 +17,25 @@ import {
 
 const useBounds = createMemo(getBounds);
 const useValidDomainForScale = createMemo(getValidDomainForScale);
+
+export const useCombinedDomain = createMemo(getCombinedDomain);
+export const useValueToIndexScale = createMemo(getValueToIndexScale);
 export const useAxisDomain = createMemo(getAxisDomain);
 
 export function useDomain(
   valuesArray: AnyNumArray,
   scaleType: ScaleType = ScaleType.Linear,
   errorArray?: AnyNumArray
-) {
+): Domain | undefined {
+  // Distinct memoized calls allows for bounds to not be recomputed when scale type changes
   const bounds = useBounds(valuesArray, errorArray);
   return useValidDomainForScale(bounds, scaleType);
 }
 
-export function useFrameRendering(): void {
-  const [, setNum] = useState(0);
-
-  useFrame(() => {
-    setNum(Math.random());
-  });
-}
-
 export function useDomains(
-  valuesArrays: (NdArray<number[]> | number[])[],
+  valuesArrays: AnyNumArray[],
```

Comment:
This hook wasn't the one that was used, so it didn't get the typed arrays changes.